apps/gamify/api/project.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Info: including the 'project.json' file in the project itself, is the new 'nx best way to do it'. So great it's here :-)

For some generators, when creating an app or lib, you have to tick on this option. So this is just a heads-ups to be aware of it.